admin_guide/overcommit.adoc

```diff
@@ -74,7 +74,23 @@ filter-pre-mux-client.conf:
   buffer_path `/var/lib/fluentd/buffer-mux-client`
 ----
 
-The Fluentd `buffer_queue_limit` is 32.
+The Fluentd `buffer_queue_limit` is the value of the variable `BUFFER_QUEUE_LIMIT`. This value is `32` by default.
+
+The environment variable `BUFFER_QUEUE_LIMIT` is calculated as `(FILE_BUFFER_LIMIT / (number_of_outputs * BUFFER_SIZE_LIMIT))`.
+
+In this formula, `number_of_outputs` is `1` if all the logs are sent to a single resource and it is incremented by `1` for each additional resource. For example, the value of `number_of_outputs` will be:
+
+ * `1` - If all logs are sent to a single ElasticSearch node.
```

Comment:
This is a terminology issue, so I might be wrong.  But when I hear an "ElasticSearch node", I have an impression the ElasticSearch is running on a node, not in a container...  In the Common Logging EFK stack, each component is containerized.  In such case, how do you usually name it?

Reply:
Thanks @nhosoi for looking at this. I will update this to `pod`. Which is consistent with rest of our docs.